_posts/fork/2021-12-21-Deepash.md

```diff
@@ -13,7 +13,7 @@ typora-copy-images-to: ../../media/theme/deepash
 
 # Deepash
 
-Base on official Ash theme,but add some vibrant colors.
+Base on official [Ash](https://github.com/typora/typora-ash-theme) theme,but add some vibrant colors.
```

Comment:
Sorry, actually I mean add a link on Ash page to your theme. like https://github.com/typora/typora-theme-gallery/pull/182/files

And, btw, it's better to add whitespace after "," (`,but` → `, but`)